docs/user-guide/src/quick-start.md

```diff
@@ -0,0 +1,642 @@
+        protocol: TCP
+      # Inspector API
+      - containerPort: 5050
+        hostPort: 5050
+        listenAddress: "0.0.0.0"
+        protocol: TCP
+```
+
+As you can see, it has a few ports forwarded from the host. This is to make
+Ironic reachable when it is running inside the kind cluster.
+
+Now go ahead and create the cluster:
+
+```bash
+kind create cluster --config kind.yaml
+```
+
+We will need to install cert-manager also. It will be used to manage the
+certificates for Ironic later.
+
+```bash
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.14.3/cert-manager.yaml
+```
+
+### DHCP server
+
+The BareMetalHosts must be able to call back to Ironic when going through the
+inspection phase. This means that they must have IP addresses in a network where
+they can reach Ironic. We will set up a DHCP server for this purpose.
+
+Any DHCP server can be used for this. We will here use the ironic container
+image that incudes dnsmasq and some scripts for configuring it.
+
+Create a configuration file like this and save it as `dnsmasq.env`:
+
+```bash
+# Specify the MAC addresses (separated by ;) of the hosts we know about and want to use
+DHCP_HOSTS=80:c1:6e:7a:e8:10;80:c1:6e:7a:5a:a8
+# Ignore unknown hosts so we don't accidentally give out IP addresses to other hosts in the network
+DHCP_IGNORE=tag:!known
+# Listen on this IP (management computer)
+PROVISIONING_IP=192.168.0.150
+# Give out IP addresses in this range
+DHCP_RANGE=192.168.0.100,192.168.0.149
+```
+
+You can now run the DHCP server like this:
+
+```bash
+docker run --name dnsmasq --rm -d --net=host --privileged --user 997:994 \
+  --env-file dnsmasq.env --entrypoint /bin/rundnsmasq \
+  quay.io/metal3-io/ironic
+```
+
+### Image server
+
+In order to do anything useful, we will need a server for hosting disk images
+that can be used to provision the servers.
+
+Create a folder to hold the disk images:
+
+```bash
+mkdir disk-images
+```
+
+Download an image to use for testing:
+
+```bash
+wget -P disk-images https://github.com/cirros-dev/cirros/releases/download/0.6.2/cirros-0.6.2-x86_64-disk.img
+```
+
+Run a basic http server to expose the disk images:
+
+```bash
+docker run --name image-server --rm -d -p 80:8080 \
+  -v "$(pwd)/disk-images:/usr/share/nginx/html" nginxinc/nginx-unprivileged
+```
+
+### Deploy Ironic
+
+In this section we will create a
+[kustomization](https://kubectl.docs.kubernetes.io/references/kustomize/glossary/#kustomization)
+containing configuration and credentials for deploying Ironic.
+
+Create a folder to hold the kustomization:
+
+```bash
+mkdir ironic
+```
+
+#### Authentication configuration
+
+Create authentication configuration for Ironic and Inspector. You will need to
+generate a username and password for each. We will here refer to them as
+`IRONIC_USERNAME`, `IRONIC_PASSWORD`, `INSPECTOR_USERNAME` and
+`INSPECTOR_PASSWORD`.
+
+Create a file `ironic-auth-config` with configuration for how to access Ironic.
```

Comment:
This all will be redundant without inspector. Since this guide is still a draft, maybe avoid inspector already? You can set USE_IRONIC_INSPECTOR=false for now.

Reply:
Unfortunately it turned out to be more work removing inspector everywhere then just using it as is. All the manifests still include and expect Inspector to be there in v0.5.0. I will happily update this once we have a new release that doesn't use Inspector though.
(I hope we can get this guide merged in time for KubeCon :crossed_fingers: )